vae_medmnist/evaluation/evaluation.py

```diff
@@ -172,11 +190,13 @@ def save_reconstructions(model, dataloader, device, save_path, descriptions):
         model = ResNetVAE.load_from_checkpoint(f"{hparams['checkpoint_dir']}/best-checkpoint.ckpt")
     elif hparams['model'] == 'cvae':
         model = CVAE.load_from_checkpoint(f"{hparams['checkpoint_dir']}/cvae-best-checkpoint.ckpt")
+    elif hparams['model'] == 'beta_vae':
+        model = BetaVAE.load_from_checkpoint(f"{hparams['checkpoint_dir']}/betavae-best-checkpoint-v4.ckpt")
```

Comment:
I'd rather you'd only use one model as before and dont add version number to file name. If you create a vX folder, you should only keep the best model anyway